classes/02class/exercises/c02-network08/andreazzabr/C02-network08.txt

```diff
@@ -0,0 +1,25 @@
+C02-network08
+
+Network ACL (NACL's)
+Look for the network ACL created for your VPC and answer the following:
+Your ACL rule inbound and outbound should look like this:
+
+Rule#    Type    Protocol    Port Range    Source    Allow/Deny
+100    ALL Traffic    ALL    ALL    <vpc-network>    ALLOW
+101    ALL Traffic    ALL    ALL    0.0.0.0/0    ALLOW
+*    ALL Traffic    ALL    ALL    0.0.0.0/0    DENY
+
+What is the difference between inbound x outbound?
+Inbound rules filters connections TO the host and outbound is FROM the host
```

Comment:
ACL is at the subnet level, so it will filter connections to/from the subnet and not to/from the host.
At the host level we use the Security Groups that are associated with network interfaces allocated to hosts.